mojio_sdk/api.py

```diff
@@ -48,6 +49,28 @@ def get_vehicles(self):
         data = response.json()
         return self._parse_devices(data)
 
+    def get_trips(self):
+        if self._authentication is None or not self._authentication.is_valid():
+            self.login()
+
+        data_url = "https://" + self._tenant + "-api.moj.io/v2/trips"
```

Comment:
Also populating the url could be a separate function. That takes the endpoint (`vehicles` or `trips`) as argument.